src/QsCompiler/LlvmBindings/DebugInfo/DebugBasicType.cs

```diff
@@ -28,20 +28,19 @@ public class DebugBasicType
     {
         /// <summary>Initializes a new instance of the <see cref="DebugBasicType"/> class.</summary>
         /// <param name="llvmType">Type to wrap debug information for</param>
-        /// <param name="module">Module to use when constructing the debug information</param>
+        /// <param name="dIBuilder"><see cref="DebugInfoBuilder"/> to use when constructing debug info</param>
         /// <param name="name">Source language name of the type</param>
         /// <param name="encoding">Encoding for the type</param>
-        public DebugBasicType(ITypeRef llvmType, BitcodeModule module, string name, DiTypeKind encoding)
+        public DebugBasicType(ITypeRef llvmType, DebugInfoBuilder dIBuilder, string name, DiTypeKind encoding)
             : base(
                 llvmType,
-                module
-                          .DIBuilder
+                dIBuilder
                           .CreateBasicType(
```

Comment:
Nit: this indentation is not consistent with the rest of the code now that the `module` part is no longer there.